astroquery/ipac/irsa/most/core.py

```diff
@@ -416,7 +416,7 @@
     'https://irsa.ipac.caltech.edu/workspace/TMP_X69utS_13312/MOST/pid15792/fitsimage_A850RA.tar.gz'
     >>> matched["region_tarball"]  # doctest: +IGNORE_OUTPUT
     'https://irsa.ipac.caltech.edu/workspace/TMP_X69utS_13312/MOST/pid15792/ds9region_A850RA.tar'
-    >>> matched["results"].columns
+    >>> matched["results"].columns  # doctest: +IGNORE_OUTPUT
```

Comment:
This doesn't trick the linter, so I would remove the commit, but then the linter won't run on the rst files either.

(otherwise the doctest infra is not sensitive to whitespace, so you can cut up the output into two lines)